project_task_code/tests/test_project_task_code.py

```diff
@@ -38,3 +38,14 @@ def test_name_get(self):
         )
         result = project_task.name_get()
         self.assertEqual(result[0][1], "[%s] Task Testing Get Name" % code)
+
+    def test_name_search(self):
+        number_next = self.task_sequence.number_next_actual
+        code = self.task_sequence.get_next_char(number_next)
+        project_task = self.project_task_model.create(
+            {
+                "name": "Task Testing Name Search",
+            }
+        )
+        result = project_task.name_search(name=code)
+        self.assertEqual(result[0][1], project_task.display_name)
```

Comment:
I'd check 1st position (ID), because we're not validating name in this test but the gotten result matches our task.